### DISCUSSION TITLE
play video reverse in react-native-video

### DISCUSSION THREAD

**Discussion by lokesh-selvam10** [TheWidlarzGroup/react-native-video#3305] [Q&A] [answered]
### What are you trying to do?

Play a video in reverse. Are there any configurable props to play the video in reverse?

**Comment by freeboub** [comment, marked as answer]
Not supported on exoplayer:
https://github.com/google/ExoPlayer/issues/3296

A better explanation of why it is complicated to implement:
https://stackoverflow.com/questions/59882306/exoplayer-2-play-video-in-reverse
